openrtb_ext/bidders.go

```diff
@@ -552,11 +556,11 @@ var bidderNameLookup = func() map[string]BidderName {
 		lookup[bidderNameLower] = name
 	}
 	return lookup
-}()
+}
```

Comment:
mostly for my understanding, any specific reason to make this change?

Reply:
because of - https://github.com/prebid/prebid-server/pull/3055#discussion_r1314615630